Source/Scene/Cesium3DTileset.js

```diff
@@ -68,10 +74,19 @@ define([
         }
         //>>includeEnd('debug');
 
-        url = appendForwardSlash(url);
+        var tilesJson;
+        if (getExtensionFromUri(url) === 'json') {
+            tilesJson = url;
+            url = getBaseUri(url);
+        } else {
+            if (url.lastIndexOf('/') !== (url.length - 1)) {
+                url = appendForwardSlash(url);
+            }
```

Comment:
You don't need to check for the trailing slash here because `appendForwardSlash` should handle that.